src/saml20_clj/sp.clj

```diff
@@ -63,13 +60,15 @@
           [:ds:KeyInfo  {:xmlns:ds  "http://www.w3.org/2000/09/xmldsig#"}
            [:ds:X509Data
             [:ds:X509Certificate certificate-str]]]]
-         [:md:SingleLogoutService  {:Binding  "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST", :Location  "https://example.org/saml/SingleLogout"}]
```

Comment:
It looks like the `SingleLogoutService` metadata entry has been removed here. Was it intended?

Reply:
That's change of @tvirolai, I don't care either way. Added back.